### ISSUE TITLE
After 1000 epoch, the training loss and test accuracy remains the same?

### ISSUE THREAD

**Issue by qm-intel** [zhichunguo/Meta-MGNN#6]
Hi @zhichunguo 

I am facing a challenge with MetaMGNN. I trained the model for 1000 epoch, but the loss and accuracy remains the same. This means it is not training the meta model at all. I am applying your code on my data with 79 node features, however, I tried with your papers features too and the result is similar.  I am wondering I am doing mistakes somewhere....

```
DEBUG:root:Epoch: 0	Train Loss: 0.8704835772514343	Test Accuracy: 43.87436384313247	
DEBUG:root:Epoch: 1	Train Loss: 0.32417869567871094	Test Accuracy: 49.572593688964844	
DEBUG:root:Epoch: 2	Train Loss: 0.09464379400014877	Test Accuracy: 42.63082504272461	
DEBUG:root:Epoch: 3	Train Loss: 0.07533042877912521	Test Accuracy: 54.545582715202784	
DEBUG:root:Epoch: 4	Train Loss: 0.11637484282255173	Test Accuracy: 44.320564943201404	
DEBUG:root:Epoch: 5	Train Loss: 0.034378036856651306	Test Accuracy: 54.640758626601276	
DEBUG:root:Epoch: 6	Train Loss: 0.03435908630490303	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 7	Train Loss: 0.03385530039668083	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 8	Train Loss: 0.03292354196310043	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 9	Train Loss: 0.0315357968211174	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 10	Train Loss: 0.03256123140454292	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 11	Train Loss: 0.03294479474425316	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 12	Train Loss: 0.032359760254621506	Test Accuracy: 50.21079287809484	
.
.
.

DEBUG:root:Epoch: 994	Train Loss: 0.031872864812612534	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 995	Train Loss: 0.03162331134080887	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 996	Train Loss: 0.03111858479678631	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 997	Train Loss: 0.03308967500925064	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 998	Train Loss: 0.031367577612400055	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 999	Train Loss: 0.03169362619519234	Test Accuracy: 50.21079287809484	
DEBUG:root:Epoch: 1000	Train Loss: 0.0316644087433815	Test Accuracy: 50.21079287809484	
INFO:root:Training has finished.
```

What I did include the followings:
- I have extracted 79 node features and 10 edge features from my data
- I want to run on MAML firts, so I set 
```
 parser.add_argument('--add_similarity', type=bool, default=False)
    parser.add_argument('--add_selfsupervise', type=bool, default=False)
    parser.add_argument('--interact', type=bool, default=False)
```
and 
```
parser.add_argument('--num_tasks', type=int, default= 33, help='number of tasks')
parser.add_argument('--num_train_tasks', type=int, default= 16, help='number of meta-training task')
parser.add_argument('--num_test_tasks', type=int, default= 17, help='number of meta-testing tasks')
parser.add_argument('--n_ways', type=int, default=2, help='n-ways of dataset')
parser.add_argument('--m_support', type=int, default=10, help='size of support set')
parser.add_argument('--k_query', type=int, default=20, help='size of query set')
parser.add_argument('--meta_lr', type=float, default=0.001,
                    help='meta learning rate (i.e., outer lr:  beta) (default: 0.001)')
parser.add_argument('--update_lr', type=float, default=0.005,
                    help='task-specific learning rate (inner lr: alpha) (default: 0.04)')
parser.add_argument('--batch_size', type=int, default= 32,
                    help='input batch size for training (default: 32)')
parser.add_argument('--epochs', type=int, default= 1000,
                    help='number of epochs to train (default: 100)')
parser.add_argument('--lr', type=float, default=0.001,
                    help='learning rate (default: 0.001')
parser.add_argument('--lr_scale',type=float, default=0.1,
                    help='relative learning rate for feature extraction layer (default: 1)')
parser.add_argument('--decay', type=float, default=0,
                    help='weight decay (default: 0)')
parser.add_argument('--num_layer' ,type=int, default=7,
                    help='number of GNN message passing layer (default: 5)')
parser.add_argument('--emb_dim', type=int, default = 256,
                    help='embedding dimension (default: 300)', choices=[64, 300])
```

**Comment by ShaoZhenLiu** [comment]
Have you found out why? I have the same question 😭
